src/Tools/dotnet-monitor/FilteredEndpointInfoSource.cs

```diff
@@ -63,8 +63,9 @@ internal class FilteredEndpointInfoSource : IEndpointInfoSourceInternal
                         _runtimeInstanceCookieToFilterOut = runtimeInstanceCookie;
                     }
                 }
-                catch (Exception)
+                catch (Exception ex)
                 {
+                    clientSourceLogger.LogTrace(ex.Message);
```

Comment:
Debug level. Also, add the LoggerExtensions instead of calling ILogger directly. And pass the exception through the exception parameter.